tests/test_packages/test_skills/test_simple_seller/test_strategy.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains the tests of the strategy class of the simple seller skill."""
+
+import logging
+from pathlib import Path
+from typing import cast
+from unittest.mock import patch
+
+import pytest
+
+from aea.test_tools.test_skill import BaseSkillTestCase
+
+from packages.fetchai.skills.simple_seller.strategy import Strategy
+
+from tests.conftest import ROOT_DIR
+
+
+class TestStrategy(BaseSkillTestCase):
+    """Test Strategy of simple seller."""
+
+    path_to_skill = Path(ROOT_DIR, "packages", "fetchai", "skills", "simple_seller")
+
+    @classmethod
+    def setup(cls):
```

Comment:
See above re test class

Reply:
This skill only has one class (strategy) to set up and test.